add_pypi_package.sh

```diff
@@ -73,12 +74,16 @@ add_to_tito_props() {
 }
 
 add_pypi_to_comps() {
-  local comps_scl="nonscl"
   local comps_packages=$(rpmspec --query --builtrpms --queryformat '%{NAME}\n' $PACKAGE_DIR/*.spec)
-  if [[ $TITO_TAG == katello-* ]]; then
+  if [[ $TITO_TAG == katello-*-pulpcore-* ]]; then
+    local comps_file="katello-pulpcore-server"
```

Comment:
Will we also have a `pulpcore-client`?

Reply:
no, this was mainly following the fact that all katello-related comps are called -server

Reply:
@pcreech any naming preferences from your POV? I think droping `-server` is the right way?

Reply:
Kill "-server" if it's not needed